### PR TITLE
Qa scenario fixes

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-2a-sbd-kvm.yaml [hunk 1]
```diff
@@ -259,8 +259,6 @@ proposals:
       ceilometer-agent-hyperv: []
       ceilometer-polling:
       - cluster:services
-      ceilometer-polling:
```
Comment (aplanas) [inline scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-2a-sbd-kvm.yaml]: I am thinking, is this duplicate entry some issue in the crowbar batch export?

**Comment by aplanas** [comment]
+1

**Comment by abel-navarro** [comment]
+1

File: scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-2a-sbd-kvm.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-2a-sbd-kvm.yaml]: no, it was just manual patching mistake.

**Pull request merged**